### PR TITLE
[Python] Added window map example, modified import statements in submodule

### PR DESCRIPTION
**Pull request by jasminevle**

### CODE REVIEW

File: python/erdos/operators/map.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import erdos
+
```
Comment (pschafhalter) [inline python/erdos/operators/map.py]: add another newline (yapf should do this for you).

File: python/erdos/operators/window.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 and have a timestamp corresponding to the most recent Message in the list.
 """
 
-import erdos.Operator
+import erdos
```
Comment (pschafhalter) [inline python/erdos/operators/window.py]: another newline here too please

File: python/examples/window_map.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+"""Windows messages from one sender based on user-defined window_size.
+Separate operator applies a map function on sent windows.
+"""
+
+from erdos.operators import window, map
+import erdos
+import time
```
Comment (pschafhalter) [inline python/examples/window_map.py]: Reformat the imports like this:

```python
import time

import erdos
from erdos.operators import window, map
```

File: python/examples/window_map.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+                    name=self.config.name, watermark=watermark))
+                self.write_stream.send(watermark)
+
+            count += 1
+            time.sleep(1 / self.frequency)
+
+def main():
+    """Creates and runs the dataflow graph."""
+
+    def add(Msg):
```
Comment (pschafhalter) [inline python/examples/window_map.py]: Rename to `msg`. Arguments shouldn't be capitalized.

File: python/examples/window_map.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        """Mapping Function passed into MapWindowOp,
```
Comment (pschafhalter) [inline python/examples/window_map.py]: Update the docstring, I think it's out of date

File: python/examples/window_map.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+    (source_stream, ) = erdos.connect(SendOp,
+                                    erdos.OperatorConfig(name="SendOp"),
+                                    [],
+                                    frequency=1)
+    (window_stream, ) = erdos.connect(window.WatermarkWindow, erdos.OperatorConfig(),
+                                    [source_stream])
+    (map_stream, ) = erdos.connect(map.Map, erdos.OperatorConfig(),
+                                    [window_stream], function=add)
+    erdos.run()
```
Comment (pschafhalter) [inline python/examples/window_map.py]: Can you add an `ExtractStream` and print out the received messages? You'll also need to `erdos.run_async()` instead.

File: python/examples/window_map.py [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+    def run(self):
+        count = 0
+        while True:
+            timestamp = erdos.Timestamp(coordinates=[count])
+            msg = erdos.Message(timestamp, count)
+            print("{name}: sending {msg}".format(name=self.config.name,
+                                                 msg=msg))
+            self.write_stream.send(msg)
+
+            if count % 3 == 0:
```
Comment (pschafhalter) [inline python/examples/window_map.py]: Remove this if statement and its contents.

File: python/examples/window_map.py [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+        total = 0
+        for msg in Msg.data:
+            total += msg.data
+        return erdos.Message(Msg.timestamp, total)
+
+    (source_stream, ) = erdos.connect(SendOp,
+                                    erdos.OperatorConfig(name="SendOp"),
+                                    [],
+                                    frequency=1)
+    (window_stream, ) = erdos.connect(window.WatermarkWindow, erdos.OperatorConfig(),
```
Comment (pschafhalter) [inline python/examples/window_map.py]: Use a `TumblingWindow` and set `window_size` to 3.

File: python/examples/window_map.py [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+           returns sums all the messages' data."""
+        total = 0
+        for msg in Msg.data:
+            total += msg.data
+        return erdos.Message(Msg.timestamp, total)
+
+    (source_stream, ) = erdos.connect(SendOp,
+                                    erdos.OperatorConfig(name="SendOp"),
+                                    [],
+                                    frequency=1)
```
Comment (pschafhalter) [inline python/examples/window_map.py]:
```suggestion
                                    frequency=3)

```

File: python/erdos/operators/map.py (hunk 1 above)
Comment (jasminevle) [reply python/erdos/operators/map.py]: ran yapf on all three files

**Pull request merged**